### DISCUSSION TITLE
本身就是政治的问题，不讨论政治不是扯淡吗

### DISCUSSION THREAD

**Discussion by Mmmmeruem** [WeNeedHome/SummaryOfLoanSuspension#844] [Show and tell]
政治在[布莱克维尔政治思想百科全书]中的概念是：一群在观点和利益方面本来很不一致的人们做出集体决策的过程。这个决策一般被认为对这个群体具有约束力并作为公共政策加以实施。

“政治是对社会价值的权威分配。”

Politics – The process by which individual interests (desires, wants, etc.) are reconciled with collective action and the provision of public goods. Interests means the things that you want, the desire that you have.

或许未必和党有关，但房屋烂尾停贷一定与政治有关

**Comment by w814929152** [comment]
在中国，一切的问题都是政治问题。

**Comment by Accelerator6666** [comment]
没法解决或者不想解决的问题统称为政治问题。

**Comment by WXWXwqz** [comment]
> 政治在[布莱克维尔政治思想百科全书]中的概念是：一群在观点和利益方面本来很不一致的人们做出集体决策的过程。这个决策一般被认为对这个群体具有约束力并作为公共政策加以实施。
> 
> “政治是对社会价值的权威分配。”
> 
> Politics – The process by which individual interests (desires, wants, etc.) are reconciled with collective action and the provision of public goods. Interests means the things that you want, the desire that you have.
> 
> 或许未必和党有关，但房屋烂尾停贷一定与政治有关

在中国，政治问题能跟党无关？？？

**Comment by yanwuguangjun** [comment]
可是问题就是 我们不能讨论政治。或者是讨论了也没有用。只能讨论解决问题的其他可能性。

**Comment by leanagainstwall** [comment]
一党专政，遍地是灾

**Reply by John-zhan** [reply]
> 一党专政，遍地是灾

还不是一党专政的事儿，是普通纳税人压根没有选票，更别提被选举权了

**Comment by benfre** [comment]
要注意不同网络平台的目的。

我相信本repo确实起到数据汇总、数据汇总和展示的工具开发的作用。

如果你有关于哪些数据的完整性、真实性相关的问题，或者对数据处理的过程脚本有疑问，可以提出来讨论。

和这个repo作用无关的内容，虽然可能涉及这些数据背后的问题，确实不是在这个平台讨论的。

**Comment by toxin08x** [comment]
建议再找个论坛去讨论深层次的政治问题吧。如果肉身在墙内并且谈论得足够过火的话，可能会给自己或者GitHub组织者们招来无妄之灾。

**Comment by baoang** [comment]
如果是否“政治”问题的划线是由国内官方定的，那么谈这个数据的问题而有意划开相关可能涉及到的话题就未必能达到作者的原本意图，因为数据本身不是事，但数据所代表的恰恰是个事，因此我认为数据的呈现和组织，及开repo，而不是数据本身，已经是政治敏感区域了。再特意强调什么政治问题只是给自己的一点安慰。目前没出什么大事，可能风暴还在酝酿之中。
八月份看吧，我猜想这事八月份会定调下来。

**Comment by h3idan** [comment]
同样不理解，不让讨论政治这事儿。明显就是政治问题啊。还再次关闭discussion，谁关闭？创建者么？怕被请去喝茶？
